terraform/terraform_test.go

```diff
@@ -1675,10 +1675,7 @@ aws_instance.foo:
 const testTFPlanDiffIgnoreChangesWithFlatmaps = `
 UPDATE: aws_instance.foo
   lst.#:   "1" => "2"
-  lst.0:   "j" => "j"
   lst.1:   "" => "k"
-  set.#:   "1" => "1"
-  set.0.a: "1" => "1"
```

Comment:
This change follows as these values are not 
* empty
* newly computed
* modified